odoo_module_migrate/migration.py

```diff
@@ -119,6 +137,13 @@ def _get_code_from_previous_branch(self, module_name, remote_name):
                 'target': target_version,
                 'module': module_name,
             }, path=self._directory_path)
+        if self._auto_squash:
+            logger.info("AutoSquash Commit")
+            _execute_shell(
+                "GIT_EDITOR=odoo-module-squasher "
+                "GIT_SEQUENCE_EDITOR=odoo-module-squasher "
+                "git rebase  -i %(target)s" % {"target": target_version}
```

Comment:
This way, users don't need a specific instruction format configured in their Git:

```suggestion
                "git -c rebase.instructionFormat='<%%ae> %%s' rebase  -i %(target)s" % {"target": target_version}
```